### PR TITLE
Archive Notes

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (479)
Screenshot (480)

In this pull request, following features are there related to archiving : 
 - Archive a note from Homepage.
 - Delete a note from Archive Page.
 - Restore a note from Archive Page to Home Page.

### CODE REVIEW

File: src/note-components/archive/archive.jsx [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
-                <Sidebar/>
-                <div className="right-cont">
-                    Archived Notes
+const { noteState, noteDispatch } = useNote();
+const { archive } = noteState;
+const { authState } = useAuth();
+const { token } = authState;
+
+const restoreNoteFromArchive = async (item) =>{
+console.log(item._id)
```
Comment (junaidshaikh-js) [inline src/note-components/archive/archive.jsx:15]: remove console logs

File: src/note-components/archive/archive.jsx [hunk 2]
```diff
@@ -1,19 +1,79 @@
...
+try{
+    const response = await axios({
+        method:"post",
+        url:`/api/archives/restore/${item._id}`,
+        headers:{authorization:token}, 
+    });
+    console.log(response)
```
Comment (junaidshaikh-js) [inline src/note-components/archive/archive.jsx:22]: Here

File: src/note-components/archive/archive.jsx [hunk 3]
```diff
@@ -1,19 +1,79 @@
...
+});
+console.log(response)
+if(response.status === 200 || response.status === 201){
+noteDispatch({type : "DELETE_NOTE_FROM_ARCHIVE", payload : {archive : response.data.archives, trash : item}})
+}
+}
+catch (error){
+console.log(error);
+}
+}
```
Comment (junaidshaikh-js) [inline src/note-components/archive/archive.jsx:48]: The spacing in the code lines should be consistent across the file. 

Use some code formatter to automate the task.

**Review by junaidshaikh-js — APPROVED** [review]
Good work done. 

Overall code looks clean. 

Make sure you remove console logs before pushing the code and also keep an eye on code formatting.

**Review by Tanay-J — APPROVED** [review]
Good work. As Junaid suggested, please use a code formatter to enhance readability.
Approving the changes.

File: src/note-components/archive/archive.jsx (hunk 3 above)
Comment (Jia-A) [reply src/note-components/archive/archive.jsx:48]: Okay will do that. I actually formatted it as HTML, maybe that is why it is presented like this.

**Pull request merged**